shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/test/java/org/apache/shardingsphere/data/pipeline/core/fixture/MigrationJobAPIFixture.java

```diff
@@ -146,6 +148,11 @@ public MigrationJobConfiguration getJobConfiguration(final String jobId) {
         return null;
     }
     
+    @Override
+    public TaskConfiguration buildTaskConfiguration(final MigrationJobConfiguration jobConfig, final int jobShardingItem, final PipelineProcessConfiguration pipelineProcessConfig) {
+        return null;
```

Comment:
return null is not a good habit

Reply:
It's in traditional Fixture code, this method won't be invoked.
And this class might be removed later, it doesn't benefit for real running.